src/send-request.js

```diff
@@ -72,6 +72,15 @@ export const xhr = ({ url, data, headers, options, captureMetrics, callback, ret
             } else {
                 const error = 'Bad HTTP status: ' + req.status + ' ' + req.statusText
                 console.error(error)
+                if (window.Sentry && window.location.host === 'app.posthog.com') {
```

Comment:
It's always wise to prepend any usage of `window` with `typeof window !== 'undefined'`, as this code might run in various environments (e.g. tests for clients' code) where the existence of `window` is not guaranteed. It'll just throw bad errors then.

Reply:
We could add `posthog.com` and www.posthog.com` (so perhaps just `*.posthog.com`) to this list.

Reply:
I can remove this entirely now 🤘